core/embed/trezorhal/stm32f4/supervise.c

```diff
@@ -5,16 +5,26 @@
 #include "../mpu.h"
 #include "common.h"
 #include "display.h"
+#include "irq.h"
 #include "supervise.h"
 
 #ifdef ARM_USER_MODE
 
+void svc_init(void) {
+  NVIC_SetPriority(SVCall_IRQn, IRQ_PRI_LOWEST);
+
+  // We need to ensure that SysTick has the expected priority.
+  // The SysTick priority is configured in the boardloader,
+  // and some early versions didn't set this properly.
+  NVIC_SetPriority(SysTick_IRQn, IRQ_PRI_HIGHEST);
```

Comment:
this may not belong here - we might also want to set this for bootloader to avoid potentially different behavior between fw and bootloader - as this affects more than SVC. how about introducing irq_init() and putting that into bootloader too?

Reply:
Discussed elsewhere: lets keep this for timer implementation.